### PR TITLE
docs: Fixing small mistakes in tutorial

### PR DESCRIPTION
**Pull request by jadarlin**

### CODE REVIEW

File: docs/source/tutorial.md [hunk 1]
```diff
@@ -104,13 +104,13 @@ print(cli.CAs)  # ca nodes in the network
 
 ### 1.2 Prepare User Id (Optionally)
 
-SDK need the credential file as a valid network user.
+The SDK needs the credentials file as a valid network user.
```
Comment (yeasy) [inline docs/source/tutorial.md:107]: credential files?

File: docs/source/tutorial.md [hunk 2]
```diff
@@ -104,13 +104,13 @@ print(cli.CAs)  # ca nodes in the network
...
 
 Typically there are two ways: using cryptogen or using Fabric-CA. That will depend on how your network boots up with.
 
-#### 1.2.1 Using Local Credential
+#### 1.2.1 Using Local Credentials
 
-SDK will load the valid credential from local path (the credentail files must be put there in advance).
+The SDK will load the valid credentials from local path (the credentials file must be put there in advance).
```
Comment (yeasy) [inline docs/source/tutorial.md:113]: credential files?

**Comment by ryjones** [comment]
Not sure why a CI failure for a doc change happened. @hyperledger/fabric-sdk-py-maintainers ?

**Pull request merged**